apps/prairielearn/src/server.ts

```diff
@@ -1736,6 +1736,13 @@ export async function initExpress(): Promise<Express> {
       coreElements: false,
     }),
   );
+  app.use(/^(\/pl\/public\/course_instance\/[0-9]+\/assessment\/[0-9]+)\/?$/, (req, res, _next) => {
+    res.redirect(`${req.params[0]}/questions`);
+  });
```

Comment:
I just added this redirect, which is exactly the same as the one on the normal instructor assessment page, because it makes the construction of the url for the assessment badge simpler/more consistent.